windows-iotcore/manage-your-device/CommandLineUtils.md

```diff
@@ -10,7 +10,10 @@ keywords: windows iot, command line, command line utilities, PowerShell
 
 # Windows 10 IoT Core Command Line Utils
 
-Looking to configure some of the settings on your device? The below tools all come pre-installed on your device to help you with that. Use PowerShell to run these commands after [connecting to your device](../connect-your-device/PowerShell.md).
+Looking to configure some of the settings on your device? The below tools are available at your disposable. Use PowerShell to run these commands after [connecting to your device](../connect-your-device/PowerShell.md).
```

Comment:
nit: "at your disposal"?

Reply:
Yes! Not sure what I was thinking there. Thanks!